### PR TITLE
Extend Avocent Support

### PR DESCRIPTION
**Pull request by electrocret**
Please give a short description what your pull request is for

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [x] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [x] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: includes/definitions/discovery/avocent.yaml [hunk 1]
```diff
@@ -1,6 +1,11 @@
-mib: PM-MIB
+mib: PM-MIB:ACS-MIB
 modules:
     sensors:
+        pre-cache:
+            data:
+                -
+                    oid:
+                        - acsSerialPortTableName
```
Comment (murrant) [inline includes/definitions/discovery/avocent.yaml:8]: I suggest caching ACS-MIB::acsSerialPortTableName instead.  Reduces issues with mib search.  (You'll have to update the usage below too.

Comment (electrocret) [reply includes/definitions/discovery/avocent.yaml]: Done

**Comment by electrocret** [comment]
Updated existing tests for ACS6048

**Comment by murrant** [comment]
Question.  If you deleted the previous test case, how do we know you didn't break it or what changed?

**Comment by electrocret** [comment]
None of the existing YAML was changed, so no less data would be collected, and both tests were performed on an ACS6048, so existing data points should be there, just different values cause they've been configured/used differently by the user. Would we prefer to have multiple tests for the same device?

**Comment by murrant** [comment]
No and yes. More variants = tests take more time for possibly no gain.
But it is extremely difficult if not very time consuming to determine if the new test data actually covers all the same cases as the existing data.

One way to do it would be to separate the code and tests data consolidation.  That way with no code changes, the new test data is more likely to be correct.

Minimizing changes in the snmprec can be another good way to reduce noise.  But has most of the same issues and could be just invalid data with the other data.

**Comment by murrant** [comment]
FYI, this PR is likely fine, just taking a second to reflect on this.

**Comment by electrocret** [comment]
@murrant Bump?

**Review by murrant — APPROVED** [review]

**Pull request merged**